### DISCUSSION TITLE
OnFunctionError Not triggered

### DISCUSSION THREAD

**Discussion by Sravanthi-Bandaru** [microsoft/semantic-kernel#9694] [Q&A] [answered]
Hi Team,
OnFunctionError is configured in the code using the process framework. When trying to execute with deliberate null exception or IO Exception, configured error function is not triggered. Can you please help with more details for this

**Comment by alliscode** [comment, marked as answer]
Hi @Sravanthi-Bandaru could you please take a look at this test and see if it helps: 

https://github.com/microsoft/semantic-kernel/blob/main/dotnet/src/Experimental/Process.IntegrationTests.Shared/ProcessTests.cs#L225
